### ISSUE TITLE
Screen reader does not announce the associate text message for the delete confirmation.

### ISSUE THREAD

**Issue by javierbo1989** [paperbits/paperbits-demo#210]
User Experience:
Users dependent on screen-reader to navigate the page will face difficulty in navigating the page if the screen reader does not announce the confirmation text information for the delete section, hence user will face difficulty in navigating the page efficiently

Repro-Steps:
1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode.
4. Navigate to the Left nav and select Pages button.
5. Navigate to the "Pages" window and select "Add page" button.
6. Select delete page button.
7. A confirmation dialogs will appear on the page.
8. Verify whether, screen reader announces the confirmation text or not.

Actual Result:
Screen reader does not announce the associate text message for the delete confirmation.
When focus reach out to the delete pop-up then, Narrator does not announce the confirmation text like, "Are you want to sure to delete". It is announcing as yes button.

Expected Result:
Screen reader should announce the associate text message for the delete confirmation as, "Are you want sure to delete 'Yes' button or No Button".
Ex: This can we have done from "aria-describedby="
This link might help: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby#example

**Screenshots**
![image](https://user-images.githubusercontent.com/88548172/204923671-32b2eac3-4f85-4d2f-996b-cf84f274175f.png)

MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

Reference Links:
[Accessibility ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0)Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
External Bug Process: If this bug belongs to external team, mark it resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDUwYzk1OWQ5MmNjZDRmZDA5MzBkYzA5OGQ5YzYzNmZhfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3MjkxNTE1NDY2MjEyMTl8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=VXJwZXd5MnZRcEc0L0lIV1dBV0gvNThzNEt2MmphN2hlR2VOQ09JYzd1MD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-cthakur%40microsoft.com)sharepoint.com) “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
Please reach out to [C&AI Teams ](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0)channel for any process related queries.

**Comment by msftedad** [comment]
Hi Team
any update on this ticket?
